inst/templates/ad_adlb.R

```diff
@@ -90,7 +90,7 @@ adlb <- lb %>%
   derive_vars_dt(
     new_vars_prefix = "A",
     dtc = LBDTC,
-    flag_imputation = FALSE
+    flag_imputation = "none"
```

Comment:
I think this line should be removed such that the default (`"auto"`) is used. This ensures compliance with ADaM IG.